datadog_checks_base/datadog_checks/base/checks/openmetrics/v2/base.py

```diff
@@ -57,7 +57,10 @@ def check(self, _):
             self.log.info('Scraping OpenMetrics endpoint: %s', endpoint)
 
             with self.adopt_namespace(scraper.namespace):
-                scraper.scrape()
+                try:
+                    scraper.scrape()
+                except Exception as e:
```

Comment:
Would it make sense to suppress tracebacks only for HTTP errors, which we expect to happen? For example `requests.exceptions.RequestException`. Otherwise this might make this check a bit too hard to debug in the future.